src/main/java/ua/antibyte/cinema/controller/Inject.java

```diff
@@ -1,36 +1,60 @@
 package ua.antibyte.cinema.controller;
 
 import java.time.LocalDateTime;
-import org.springframework.web.bind.annotation.GetMapping;
+import java.util.Set;
+import javax.annotation.PostConstruct;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 import ua.antibyte.cinema.model.CinemaHall;
 import ua.antibyte.cinema.model.Movie;
 import ua.antibyte.cinema.model.MovieSession;
+import ua.antibyte.cinema.model.Role;
+import ua.antibyte.cinema.model.User;
 import ua.antibyte.cinema.service.CinemaHallService;
 import ua.antibyte.cinema.service.MovieService;
 import ua.antibyte.cinema.service.MovieSessionService;
-import ua.antibyte.cinema.service.security.AuthenticationService;
+import ua.antibyte.cinema.service.RoleService;
+import ua.antibyte.cinema.service.ShoppingCartService;
+import ua.antibyte.cinema.service.UserService;
 
 @RestController
 @RequestMapping("/inject")
 public class Inject {
```

Comment:
We can inject data without a controller, just using Component, Service, or Repo annotation over the class and using @PostConstruct over the method which implements injection.